src/addAndRemove.js

```diff
@@ -0,0 +1,134 @@
+export class ToDoList {
+  constructor() {
+    this.tasks = JSON.parse(localStorage.getItem('list') || '[]');
+    this.complete = false;
+    this.inputTask = document.querySelector('.inputTask');
+    this.listItems = document.querySelector('.listItems');
+    this.completedTasks = document.querySelector('.clear');
+    this.indexStart = 0;
+    this.indexDrop = 0;
+  }
+
+  listTheItems = () => {
+    this.tasks.forEach((todo, index) => {
+      const div = document.createElement('div');
+      const li = document.createElement('li');
+      li.textContent = `${todo.description}`;
+      if (todo.completed === true) {
+        li.style.textDecoration = 'line-through';
+      }
+      div.id = index;
+      div.draggable = 'true';
+      div.innerHTML = `<i class="fas fa-ellipsis-v dots"></i>
+      <input type="checkbox" class="box" ${todo.completed ? 'checked' : ''}>`;
+      div.className = 'listItem';
+      div.appendChild(li);
+      this.listItems.appendChild(div);
+    });
+  };
+
+  addtask = () => {
+    this.inputTask.addEventListener('keyup', (event) => {
+      if (event.keyCode === 13) {
+        const newTask = {
+          description: this.inputTask.value,
+          index: this.tasks.length + 1,
+          completed: false,
+        };
+        const newTaskAdded = this.tasks.concat(newTask);
+        this.localStorage(newTaskAdded);
+        this.inputTask.value = '';
+      }
+    });
+  };
```

Comment:
I recommend you add a condition to check if the input value is empty, to avoid adding an empty task to the list.

Reply:
Thank you for pointing this out :+1: